test/Java10andUp/src/org/openj9/test/stackWalker/StackWalkerTestJava10.java

```diff
@@ -22,45 +22,92 @@
 package org.openj9.test.stackWalker;
 
 import java.util.HashMap;
+import java.util.Iterator;
 import java.util.List;
+import java.util.Optional;
 import java.util.stream.Collectors;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertTrue;
+import static org.testng.Assert.fail;
+
 import java.lang.StackWalker.Option;
 import java.lang.StackWalker.StackFrame;
+import java.lang.invoke.MethodType;
 
 import org.testng.annotations.Test;
 import org.testng.log4testng.Logger;
 
-@Test(groups = { "level.extended" })
-@SuppressWarnings("nls")
+/**
+ * This test StackWalker features added in Java 10 and is supplementary to the Java 9 tests.
+ *
+ */
 public class StackWalkerTestJava10 {
 	static final String TEST_MYLOADER_INSTANCE = "TestMyloaderInstance";
 	protected static Logger logger = Logger.getLogger(StackWalkerTestJava10.class);
 
-	@Test
+	@Test(groups = { "level.extended" })
 	public void testSanity() {
-		f1(3);
+		sanityMethod_1(3);
 	}
 
-	void f1(int i) {
-		if (i > 0) f1(i-1); else f2(new Integer(42));
+	void sanityMethod_1(int i) {
+		if (i > 0) sanityMethod_1(i-1); else sanityMethod_2(new Integer(42));
 	}
 
-	private void f2(Integer in) {
-		f3(in, new HashMap<String, Integer>());
+	private void sanityMethod_2(Integer in) {
+		sanityMethod_3(in, new HashMap<String, Integer>());
 	}
 
-	private int f3(Integer in, HashMap<String, Integer> hashMap) {
+	private int sanityMethod_3(Integer in, HashMap<String, Integer> hashMap) {
 		printStack("Hello");
 		return 42;
 	}
 
 	public void printStack(String arg) {
-		List<StackFrame> frameList = (StackWalker.getInstance(Option.RETAIN_CLASS_REFERENCE))
-		.walk(s -> s.collect(Collectors.toList()));
-		String expectedDescriptors[];
-		for (StackFrame f: frameList) {
-			System.out.println(f.getDescriptor());
+		final ClassLoader myClassLoader = this.getClass().getClassLoader();
+		List<StackFrame> frameList = 
+				StackWalker.getInstance(Option.RETAIN_CLASS_REFERENCE)
+				.walk(s -> s.collect(Collectors.toList()));
+		String expectedDescriptors[] = {
+				"(Ljava/lang/String;)V",
```

Comment:
Shouldn't these strings have `//NON-NLS` comments?

Reply:
The project settings were not configured to warn about non-externalized strings.
Settings updated, comments added.